viper.go

```diff
@@ -761,7 +764,12 @@ func (v *Viper) find(key string) interface{} {
 			return cast.ToBool(flag.ValueString())
 		case "stringSlice":
 			s := strings.TrimPrefix(flag.ValueString(), "[")
-			return strings.TrimSuffix(s, "]")
+			s = strings.TrimSuffix(s, "]")
+			if len(s) == 0 {
+				return []string{}
+			}
+			res, _ := readAsCSV(s)
```

Comment:
Thanks for the feedback.  pflag does in fact handle stringSlice correctly, and it knows the actual underlying Go value, but the best I can tell, it would take changes to pflag and viper APIs to be able to get to the underlying Go value.  This PR solves the immediate problem without changing the APIs.

Having said that, I did test adding a `Get() interface{}` func to the `pflag.Value` and `viper.FlagValue` interfaces.  `Get()` returns the underlying Go value instead of string that we have to parse.  It works, but I'm unclear on the ramifications and don't feel comfortable submitting a PR at this point.